scripts/check-licenses.js

```diff
@@ -20,7 +20,7 @@ async function getLicenses() {
     licenseChecker.init(
       {
         start: path.resolve(__dirname, '..'),
-        direct: true,
+        direct: false,
```

Comment:
I guess this means we also check transitive dependencies.
The original ideas was to only check the direct dependencies.
Question: do we have to check transitive dependencies?

Reply:
this was just a test but true or false do not really make a difference. The problem is that dependencies get hoisted up in the node_modules and are found there. In the end this is not bad, because we do not get wrong licences even if they come in from an other dependency. However, if they are labled wrong this becomes an issues. In the corrent case the `python-2.0` licence is fine so I just added it.